src/nestegg.c

```diff
@@ -3102,6 +3153,78 @@ nestegg_read_packet(nestegg * ctx, nestegg_packet ** pkt)
   return 1;
 }
 
+int
+nestegg_read_last_packet(nestegg * context, unsigned int track,
```

Comment:
How important is the performance of this? Scanning through the entire stream could be a little expensive for large streams, if Cues are present we could improve this by scanning from the last marked Cluster.  That can be a future improvement if needed, doesn't need to be changed now.

Reply:
Thanks! Then I will post another PR to improve this.